docs/dev/git.md

```diff
@@ -189,11 +189,21 @@ It's very important that you **always work inside a branch**. If you make any co
    git clone https://github.com/fastai/fastai
    cd fastai
    ```
+
+   Uninstall any previously installed `fastai` library:
+
+   ```bash
+   pip uninstall fastai
+   conda uninstall fastai
```

Comment:
The guide forgot to remind people to uninstall potentially installed fastai library.

Reply:
Yes!
Though only conda needs to be uninstalled. pip will uninstall its preinstalled package when installing an editable one. See: https://docs.fast.ai/dev/develop.html#editable-install-explained

Reply:
I added a whole sub-step explaining that.

Reply:
Thanks for the explanation!